privacy notice

danielstreif opened the pull request:
<!--
Thanks for your interest in our project. Bugs filed and PRs submitted are appreciated!

Please make sure that you are familiar with and follow the Code of Conduct for
this project (found in the CODE_OF_CONDUCT.md file).

Also, please make sure you're familiar with and follow the instructions in the
contributing guidelines (found in the CONTRIBUTING.md file).

Please fill out the information below to expedite the review and (hopefully)
merge of your pull request!
-->
![image](https://user-images.githubusercontent.com/34481945/68990739-1a9e9980-0857-11ea-8a52-b39cf1dbcb28.png)

<!-- What changes are being made? (What feature/bug is being fixed here?) -->

**What**:
- Add privacy notice to support page
- Add mailchimp gdpr notice to subscribe page

<!-- Why are these changes necessary? -->

**Why**:
- Legal reasons

<!-- How were these changes implemented? -->

**How**:

<!-- Have you done all of these things?  -->

**Checklist**:

<!-- add "N/A" to the end of each line that's irrelevant to your changes -->

<!-- to check an item, place an "x" in the box like so: "- [x] Documentation" -->

- [ ] Documentation added to the
      [README](https://github.com/Lernwerkstatt/site/blob/master/README.md)
- [ ] Tests
- [x] Ready to be merged
      <!-- In your opinion, is this ready to be merged as soon as it's reviewed? -->

<!-- feel free to add additional comments -->
Fixes #196

hikes101 commented:
So far I used the same modals as for the contact page. @pheeria let me know if I should create a copy or change the name for the contact modal.

Also, right now the checkbox under the subscription form is just a dummy. It is not implemented in the form and doesn't send any data to mailchimp. @starmaerker can you help with that since you created the form?

pheeria commented:
I guess we need to implement validation in JavaScript using `disabled` attribute for both forms - mailchimp and contact.

jarjan commented:
@pheeria @hikes101 I think who have access to mailchimp can generate/get code that we can copypaste here.

hikes101 commented:
@jarjan @pheeria so the idea is to recreate the form on mailchimp and replace it?

hikes101 commented:
@pheeria @jarjan I created a new form with mailchimp and apparently they now have their own gdpr terms included. That's probably better than our disclaimer. I had to change the layout a bit since the two column setup did not work properly with that much text.

![image](https://user-images.githubusercontent.com/34481945/68990786-8d0f7980-0857-11ea-9ca4-4132be07afee.png)

danielstreif commented:
@pheeria @jarjan could you please review this pr? thanks!

pheeria commented:
Sorry, for taking so long 😞 

There are a couple of points I want to suggest:
1. Subscribe button is off of our color scheme. I suggest you use the same as in Facebook modals. (@jarjan was going to look at the design refresh some time :) )
2. `info@die-lernwerkstatt.org` after the checkmark should be wrapped into an anchor tag with `mailto:`
3. Title/Subtitle looks weird to me, but feel free to ignore this comment. We don't have to address it now
4. The whole page looks nicely in Chrome, but Firefox version looks unattractive. Now that we are followed everywhere by a Firefox ads in Berlin, we shouldn't forget to check on Firefox before we release
5. All of the text should be extracted to locales
6. `contacts.json` on Support page is not used, as far as I can see. Btw, why are our policies called contacts? :D

How it looks like in Firefox:
![image](https://user-images.githubusercontent.com/15848876/69502011-dbcdab00-0f0a-11ea-99b4-dabf4c1756c5.png)

danielstreif commented:
> Sorry, for taking so long :disappointed:
> 
> There are a couple of points I want to suggest:
> 
>     1. Subscribe button is off of our color scheme. I suggest you use the same as in Facebook modals. (@jarjan was going to look at the design refresh some time :) )
> 
>     2. `info@die-lernwerkstatt.org` after the checkmark should be wrapped into an anchor tag with `mailto:`
> 
>     3. Title/Subtitle looks weird to me, but feel free to ignore this comment. We don't have to address it now
> 
>     4. The whole page looks nicely in Chrome, but Firefox version looks unattractive. Now that we are followed everywhere by a Firefox ads in Berlin, we shouldn't forget to check on Firefox before we release
> 
>     5. All of the text should be extracted to locales
> 
>     6. `contacts.json` on Support page is not used, as far as I can see. Btw, why are our policies called contacts? :D
> 
>

@pheeria Thanks for the reply :slightly_smiling_face: 

1. Done
2. Done
3. I removed the subtitle for now, just a legacy of the old strikingly site. We can discuss about the subtitles on other pages another time.
4. You probably have your firefox security settings on "strict", so the remote css of mailchimp is blocked. I have the same issue but when you switch to "standard" it looks normal. I don't think we can do much unless we rewrite the whole mailchimp form. When you choose the "strict" setting you agree that some pages won't work properly.
5. Done
6. On Support it is used but not on Subscribe. Before the mailchimp disclaimer I had a version using our own policy modals on the Subscribe page, seems like I forgot to remove it. Done now.
I also renamed `contact.json` to `policy.json`. I think Andreas named it contact at first because the policies were only used on the Contact page.

jarjan commented on src/views/partials/subscribe/mailchimp.hbs:
```diff
@@ -1,68 +1,101 @@
-          <label for="mce-FNAME">{{{__ "common_name"}}} </label>
-          <input type="text" value="" name="FNAME" class="form-control" id="mce-FNAME">
+      <div class="mc-field-group">
+        <label for="mce-FNAME">Vorname </label>
```
```suggestion
        <label for="mce-FNAME">{{{__ "common_name"}}}</label>
```

jarjan commented on src/views/partials/subscribe/mailchimp.hbs:
```diff
@@ -1,68 +1,101 @@
+
+      <div class="indicates-required"><span class="asterisk">*</span> indicates required</div>
+      <div class="mc-field-group">
+        <label for="mce-EMAIL">E-Mail-Adresse <span class="asterisk">*</span>
```
```suggestion
        <label for="mce-EMAIL">{{{__ "common_email"}}}<span class="asterisk">*</span>
```

jarjan commented on src/views/partials/subscribe/mailchimp.hbs:
```diff
@@ -1,68 +1,101 @@
+        <input type="text" value="" name="FNAME" class="" id="mce-FNAME">
+      </div>
+      <div class="mc-field-group">
+        <label for="mce-LNAME">Nachname </label>
```
```suggestion
        <label for="mce-LNAME">{{{__ "common_name"}}}</label>
```

jarjan commented on src/views/partials/subscribe/mailchimp.hbs:
```diff
@@ -1,68 +1,101 @@
       <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text"
           name="b_63598878eb7adee616350bca7_9367cfd8d5" tabindex="-1" value=""></div>
-
+      <div class="clear"><input type="submit" value="Subscribe" name="subscribe" id="mc-embedded-subscribe"
```
```suggestion
      <div class="clear"><input type="submit" value="{{{__ "subscribe_button"}}}" name="subscribe" id="mc-embedded-subscribe"
```

jarjan commented on src/views/partials/subscribe/mailchimp.hbs:
```diff
@@ -1,68 +1,101 @@
-  * Translated default messages for the $ validation plugin.
-  * Locale: DE
-  */
-  if ('{{{__ "locale"}}}' === 'de') {
```
And probably adapt copy this check too, since here I am including translations only for deutsch version

danielstreif commented:
> :fire:
> Only few not super important i18n comments.

:+1: 
Good call, thanks! I just copied the snippet from mailchimp and didn't realize you created i18n labels before. Changed it.
